src/storage_broker/utils/config.py

```diff
@@ -24,40 +24,71 @@ def get_namespace():
     except EnvironmentError:
         logger.info("Not running in openshift")
 
+if os.getenv("CLOWDER_ENABLED") == "true":
+    from app_common_python import LoadedConfig, KafkaTopics, ObjectBuckets
+    cfg = LoadedConfig
+
 
 # Inventory
 INVENTORY_URL = os.getenv(
     "INVENTORY_URL", "http://insights-inventory:8080/api/inventory/v1/hosts"
 )
 
 # Kafka
-VALIDATION_TOPIC = os.getenv("CONSUME_TOPIC", "platform.upload.validation")
-ANNOUNCER_TOPIC = os.getenv("ANNOUNCER_TOPIC", "platform.upload.available")
-STORAGE_TOPIC = os.getenv("STORAGE_TOPIC", "platform.upload.buckit")
-EGRESS_TOPIC = os.getenv("EGRESS_TOPIC", "platform.inventory.events")
-TRACKER_TOPIC = os.getenv("TRACKER_TOPIC", "platform.payload-status")
-BOOTSTRAP_SERVERS = os.getenv("BOOTSTRAP_SERVERS", "kafka:29092").split()
+if os.getenv("CLOWDER_ENABLED") == "true":
+    VALIDATION_TOPIC = KafkaTopics["platform.upload.validation"].name
+    ANNOUNCER_TOPIC = KafkaTopics["platform.upload.available"].name
+    STORAGE_TOPIC = KafkaTopics["platform.upload.buckit"].name
+    EGRESS_TOPIC = KafkaTopics["platform.inventory.events"].name
+    TRACKER_TOPIC = KafkaTopics["platform.payload-status"].name
+    BOOTSTRAP_SERVERS = [f"{cfg.kafka.brokers[0].hostname}:{cfg.kafka.brokers[0].port}"]
```

Comment:
wonder if creating the full hostname could be a client lib function

Reply:
yeh - thought about that a bunch of times